styles/style.css

```diff
@@ -0,0 +1,12 @@
+header{
+    font-family: 'Roboto', sans-serif;
+     color:#c4c4c4;
```

Comment:
can we add the colors to a variable? You can create another files named "reset" and "variables" and import them on this file. In this reset, you can add the default reset for all elements and in the variables the css variables. Something like. You can use the lari repo as reference https://github.com/lahgomes/lahgomes.github.io/blob/main/css/reset.css

```css
* {
    box-sixing; border-box;
    margin: 0;
}
:root {
 --gray: #c4c4c4;
}
```

And in your css you add it like this

```css
color: var(--gray);
```